packages/cashier/src/pages/payment-agent/payment-agent-list/payment-agent-list.jsx

```diff
@@ -32,13 +34,25 @@ const PaymentAgentList = ({
 
     React.useEffect(() => {
         if (typeof setSideNotes === 'function' && !is_loading) {
-            setSideNotes(!is_try_withdraw_successful ? [<PaymentAgentDisclaimer key={0} />] : []);
+            setSideNotes(
+                !is_try_withdraw_successful
+                    ? [
+                          <SideNote has_title={false} key={0}>
+                              <PaymentAgentDisclaimer />
+                          </SideNote>,
+                      ]
+                    : []
+            );
```

Comment:
```suggestion
            const side_notes = [];

            if (!is_try_withdraw_successful) {
                side_notes.push(
                    <SideNote has_title={false} key={0}>
                        <PaymentAgentDisclaimer />
                    </SideNote>
                );
            }

            setSideNotes(side_notes);
```
@george-usynin-binary Better not to put logic inside the `setSideNotes` callback, It's just a suggestion tho 🙇🏻